test/against-ref.js

```diff
@@ -39,8 +39,8 @@ test('XX handshake with reference impl', t => {
     publicKey: responder.e.pub
   }
 
-  const client = ref.initialize('XX', true, Buffer.alloc(0), sClient, eClient)
-  const server = ref.initialize('XX', false, Buffer.alloc(0), sServer, eServer)
+  const client = ref.initialize('XX', true, Buffer.alloc(0), sClient)
+  const server = ref.initialize('XX', false, Buffer.alloc(0), sServer)
```

Comment:
ah didn't mean to push this @chm-diederichs but this fixes my assert failure here